### ISSUE TITLE
Handling errors missing from readme

### ISSUE THREAD

**Issue by apaatsio** [AnyFetch/restify-logger#4]
The readme examples use the `.use()` function.

``` javascript
app.use(logger())
```

This seems to log only successful requests.

**If you want to log errors** you should use `.pre()`:

``` javascript
app.pre(logger())
```

I think this should be clarified in the readme.

**Comment by Neamar** [comment]
That's right! But i think in this case you won't be able to get the request time then?
